app/vlstorage/netselect/netselect.go

```diff
@@ -230,6 +231,14 @@ func (sn *storageNode) getStreamIDs(qctx *logstorage.QueryContext, limit uint64)
 	return sn.getValuesWithHits(qctx, "/internal/select/stream_ids", args)
 }
 
+func (sn *storageNode) getTenantIDs(ctx context.Context, start, end int64) ([]byte, error) {
+	args := url.Values{}
+	args.Set("start", fmt.Sprintf("%d", start))
```

Comment:
hmm, you already send `start` and `end` args as integers from `vlselect` to `vlstorage`. Then why do you try parsing them as strings at the `vlstorage` side?